components/about_build_modal/about_build_modal.tsx

```diff
@@ -14,35 +15,41 @@ import {AboutLinks} from 'utils/constants';
 
 import AboutBuildModalCloud from './about_build_modal_cloud/about_build_modal_cloud';
 
-export default class AboutBuildModal extends React.PureComponent {
-    static defaultProps = {
-        show: false,
-    };
+type Props = {
+
+    /**
+     * Function that is called when the modal is dismissed
+     */
+    onHide: () => void;
+
+    /**
+     * Global config object
+     */
+    config: Partial<ClientConfig>;
```

Comment:
Is the only reason we're using `Partial` here to facilitate the tests?
If so, I think we should mock out the other variables in the tests and use just the base `ClientConfig` and `ClientLicense` here.

Reply:
Not really - there's code in `mattermost-redux/selectors/entities/general..d.ts` which uses a `Partial<ClientConfig>` - and I didn't want to change it in this PR.

Reply:
Huh, odd. It looks like `license` isn't using partial in `general.d.ts` though so we can change that one.

Reply:
Can that be handled in a separate PR please? I dont want to open a can of worms with that single change.

Reply:
I'm not suggesting making any changes in `mattermost-redux`, I was just saying we could change the field type from `Partial<ClientLicense>` to `ClientLicense`. Or is that not possible?

Reply:
done in the latest commit